src/controller/python/test/test_scripts/mobile-device-test.py

```diff
@@ -79,6 +79,9 @@ def main():
     FailIfNot(test.TestDiscovery(discriminator=TEST_DISCRIMINATOR),
               "Failed to discover any devices.")
 
+    FailIfNot(test.SetThreadOperationalDataset(dataset=TEST_THREAD_NETWORK_DATASET_TLV),
```

Comment:
should this be SetNetworkCommissioningParameters?

Reply:
Indeed it should. Thanks.